src/themes/theme.json

```diff
@@ -89,293 +89,293 @@
         "zep-typo--normal-h1": {
           "rem": "2.5rem",
           "px": "40px",
-          "specifications": [
-            "font-family: 'Roboto', sans-serif",
-            "font-weight: 300 (light)",
-            "line-height: 60px",
-            "text-transform: none",
-            "letter-spacing: 0.4px"
-          ]
+          "specifications": {
+            "font-family": "'Roboto, sans-serif'",
```

Comment:
the single quotes are wrongly placed, sans-serif should be outside because it's the backup font-face